synapse/storage/registration.py

```diff
@@ -92,6 +92,13 @@ def get_user_by_id(self, user_id):
             query, user_id
         )
 
+    def get_all_users(self):
+        query = ("SELECT users.name FROM users")
```

Comment:
Parentheses? Might be nice to try to only use them for tuples or multi-line text to avoid confusion.